tests/python_tests/test_vlm_pipeline.py

```diff
@@ -67,19 +67,26 @@ def streamer(word: str) -> bool:
         images = []
         for link in links:
             images.append(get_image_by_link(link))
-
-        ov_pipe.start_chat()
-
+        
+        # Test non chat mode
         result_from_streamer = []
         res = ov_pipe.generate(prompts[0], images=images, generation_config=generation_config, streamer=streamer)
         assert res.texts[0] == ''.join(result_from_streamer)
 
-        for prompt in prompts[1:]:
+        # Test chat mode
+        for system_message in ["", "You are a helpful assistant."]:
```

Comment:
Since you are waiting for my fix anyway, could you refactor the test :) `Test non chat mode` section should become a separate test and `system_message` should become a parameter for this test. That introduces a better granularity. `image_links_for_testing` shouldn't be a parameter because it tests calling start_chat() multiple times.

Reply:
Test is splitted